lib/active_model/serializer/adapter.rb

```diff
@@ -101,6 +101,10 @@ def fragment_cache(*args)
         raise NotImplementedError, 'This is an abstract method. Should be implemented at the concrete adapter.'
       end
 
+      def default_key_format
+        :unaltered
+      end
```

Comment:
I'd stick this in the config, maybe something like https://github.com/rails-api/active_model_serializers/pull/1050/files#diff-ae1e38e96a8bf28ae37918a65c448c73R7